payjoin-cli/src/app/v1.rs

```diff
@@ -147,7 +147,7 @@ impl App {
             "Listening at {}. Configured to accept payjoin at BIP 21 Payjoin Uri:",
             listener.local_addr()?
         );
-        println!("{}", pj_uri_string);
+        println!("{pj_uri_string}");
```

Comment:
This seems like an unrelated change. in the future can you please seperate out clippy fixes in their own commit.